assets/css/style.css

```
@@ -918,167 +839,189 @@ h2 a {
 
 @media (min-width: 976px) {
-        #header #menu-principal .nav-links li {
+        #header #main-menu #top-links li {
           text-align: center;
         }
       }
 
-#header #menu-principal .nav-links li {
-        font-size: 16px;
+#header #main-menu #top-links li {
+        font-size: rfs(16px);
```

Biome

[error] Unexpected unknown function: rfs

Use a known function instead.
See MDN web docs for more details.

(lint/correctness/noUnknownFunction)